thirdeye/thirdeye-frontend/app/pods/components/rootcause-chart-toolbar/component.js

```diff
@@ -44,9 +37,9 @@ export default Component.extend({
    * @type {String[]} - array of string values of options
    */
   granularityOptions: Object.keys(granularityMapping),
-
+  // eslint-disable-next-line ember/avoid-leaking-state-in-ember-objects
```

Comment:
Lets define these arrays outside the component as constants and reference them here to avoid having to bypass eslint.